src/GitHub.InlineReviews/ViewModels/CommentThreadViewModel.cs

```diff
@@ -44,6 +45,30 @@ public ReactiveCommand<ICommentModel> PostComment
             }
         }
 
+        public ReactiveCommand<ICommentModel> EditComment
+        {
+            get { return editComment; }
+            set
```

Comment:
I see you followed `PostComment` and made this setter public - I think that was an oversight, it should be `protected`. When you fix this, could you fix `PostComment` too?

Reply:
I made them private, but they aren't used. Are we just leaving them here in case we ever need them? So we know what actions should be taken on set?

Reply:
Made them `protected`